deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+              args:
+                - modules/allocation/main.py
+                - action: create
+                - provider: "{infra-provider}"
+                - size: small
+                - composite-name: "{agent}"
+                - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
+                - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          cleanup:
+            this: process
+            with:
+              path: python3
+              args:
+                - modules/allocation/main.py
+                - action: delete
+                - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          foreach:
+            - variable: agents-os
+              as: agent
+      ```
+
+      Following the schema of the example:
+
+      Configure the following parameters depending on your test case:
+
+      ```yaml
+      variables/agent-os
+      variables/manager-os
+      infra-provider
+      working-dir
+      tasks
+      ```
+
+      Pay attention to the tasks:
+
+      ```yaml
+      args
+      depends-on
+      ```
+
+      >Note: In args, configure the launcher's path correctly (main.py files in each module), and to fill `depends-on`, consider the steps of your test (allocation, provision, and test)
+
+3. Execution of Command (local):
+
+  Execute the command by referencing the parameters required by the library (launcher).
+
+  ```bash
+  python3 -m workflow_engine {.yaml fixture path}
+  ```
+
+  Example
+
+  ```bash
+  python3 -m workflow_engine modules/workflow_engine/examples/dtt1-agents-poc.yaml
+  ```
+
+  > Note The command execution can also be mediated through Jenkins.
+
+#### Manual execution of the Allocation module
+
+If one wishes to execute the allocation module without installing the workflow engine, they can proceed by using the launcher ([module/allocation/main.py](main.py)):
+
+1. Create
+
+  While in wazuh-qa/deployability
+
+- Local deployment (Vagrant)
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ vagrant }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}'
+
+  ```
+
+  Example:
+  ```bash
+  python3 modules/allocation/main.py --action create --provider vagrant --size large --composite-name linux-ubuntu-22.04-amd64 --inventory-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/inventory.yaml" --track-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/track.yaml"
+  ```
+
+- AWS deployment
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ aws }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}' --label-termination-date '{{ termination-date }}' --label-team  '{{ team }}'
+
+  ```
+
+  >Note: In the case of AWS it is mandatory to define 2 arguments that are not necessary for Vagrant, --label-termination-date and --label-team.
```

Comment:
Avoid using numbers, refer to them by their names: `two`, and apply this to the entire document

Reply:
Done a81aec813